cmake/setup_write_config.cmake

```diff
@@ -197,7 +202,6 @@ foreach(_feature ${_deal_ii_features_sorted})
     endif()
 
     if(_feature MATCHES "KOKKOS")
-      _detailed("#            KOKKOS_VERSION = ${Kokkos_VERSION}\n")
```

Comment:
You state that we already do print this information for the package; I checked some of my previous `detailed.log` and could not find another description of the version, apart from the possible library files we list elsewhere (which, in case Kokkos comes from Trilinos, is not listed separately). Does your change above somehow change this? I can't see that because you only do a renaming.

I might also be confused because 5 lines above we print the `MPI_CXX_VERSION` in addition, which in my case repeats the `MPI_VERSION` that comes from the automatic configuration variables. Wouldn't you then also want to delete line 201?

Reply:
This looks like a workaround for the case that only `Kokkos_VERSION` is available and not `KOKKOS_VERSION`.